ncm-accounts/src/main/pan/components/accounts/schema.pan

```diff
@@ -6,6 +6,22 @@ declaration template components/accounts/schema;
 
 include { 'quattor/schema' };
 
+function has_unique_attr = {
+    values = ARGV[0];
+    attr = ARGV[1];
+    cattrs = list();
```

Comment:
I am not sure about the name - is the c prefix for seen? If so I would prefer to spell it out -  "seen_attrs". I think it would also be more efficient to store this as an nlist Then the check below is just "if exists(seen_attrs[el[attr]])" instead of a search through a list.